docs/wiki/research/PBS/PTC-Specs.md

```diff
@@ -0,0 +1,484 @@
+# ePBS Design Specifications
+
+The current ePBS specification[^1][^2] addresses a critical issue in Ethereum's current implementation of PBS. Traditionally, both proposers and builders have had to rely on intermediaries through [MEV-Boost](/docs/wiki/research/PBS/mev-boost.md), which introduces trust and censorship concerns as outlined in the [ePBS document](/docs/wiki/research/PBS/ePBS.md). The ePBS specifications framework modifies this dynamic by changing the necessity of intermediaries ("must") to an option ("may"), allowing for a more trustless interaction within the Ethereum ecosystem. 
+
+## Specifications Overview
+
+The ePBS specification is divided into separate components to build on top of the existing specifications of Ethereum components. 
+- `Beacon-chain.md`: This document specifies the beacon chain specifications of the ePBS feature[^6].
```

Comment:
Referring to these documents without the actual link to spec is confusing. Since it wasn't merged yet, I would suggest including a link to a specific commit in Potuz's PR

Reply:
> Referring to these documents without the actual link to spec is confusing. Since it wasn't merged yet, I would suggest including a link to a specific commit in Potuz's PR

May be I will rename this to `ePBS-PTC-Specs.md`. Because this specs is based on PTC and at sometime we should see other constructions based on PEPC because PEPC is more general construction and removes a lot of assumptions.

The very first line has two reference links to the Potuz' commit. The specs are explained in the hackmd doc, the repo mainly contains the python specs not a lot of explanation and the flow. I will make the links more specific and link the repo branch that has the specs.